static/js/services/send-message.js

```diff
@@ -18,69 +18,123 @@ var _ = require('underscore'),
 
       'ngInject';
 
-      var createMessageDoc = function(user, recipients) {
+      var createMessageDoc = function(user) {
         var name = user && user.name;
-        var doc = {
+
+        return  {
           errors: [],
           form: null,
           from: user && user.phone,
           reported_date: Date.now(),
           tasks: [],
           read: [ name ],
+          //TODO: should we rename this to outgoing_message or something?!
```

Comment:
I'll raise a ticket about it, and link it in that TODO so we can work out when to fit it in.